client/reader/tag-stream/main.jsx

```diff
@@ -140,30 +139,41 @@ class TagStream extends Component {
 				<QueryReaderTag tag={ this.props.decodedTagSlug } />
 				<DocumentHead
 					title={ this.props.translate( '%s ‹ Reader', {
-						args: title,
+						args: this.props.title,
 						comment: '%s is the section name. For example: "My Likes"',
 					} ) }
 				/>
 				{ this.props.showBack && <HeaderBack /> }
 				<TagStreamHeader
-					title={ title }
+					title={
+						this.props.isPromptTag && this.props.description
```

Comment:
In keeping with the idea from D110121-code of making the description generic, I think you could just use:
```suggestion
						this.props.description ? this.props.description : this.props.title
```

Reply:
Good call on trying to make things generic. I removed the isPromptTag prop, but am still using a little bit of a hack inside the component for setting the correct title size before the tag is fully loaded, to save some refactoring that will only be needed if tag descriptions end up getting used for other purposes.